### PR TITLE
PositionGo extension

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: store/extensions/PositionGo/extension.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      }
+    ],
+    "version": "1.0.0",
+    "categories": ["Usability", "Others"],
+    
+    "source": "https://github.com/iUseYahoo/PositionGo",
+    
+    "framework": {
+      "name": "G-Node",
+      "version": "0.2.0"
```
Comment (WiredSpast) [inline store/extensions/PositionGo/extension.json]: You have 0.2.9 listed in your package.json
```suggestion
      "version": "0.2.9"
```

File: store/extensions/PositionGo/extension.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      {
+        "name": "iUseYahoo",
+        "hotel": ".com",
+        "username": "floppidity",
+        "discord": "Wh1z#6363"
+      }
+    ],
+    "version": "1.0.0",
+    "categories": ["Usability", "Others"],
+
```
Comment (WiredSpast) [inline store/extensions/PositionGo/extension.json:16]: Missing the required stable tag
```suggestion

    "stable": true,
    
```

File: store/extensions/PositionGo/extension.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    "commands": {
+        "default": ["java", "-jar", "NodeExtensionSetup.jar", "-e", "posgo.js", "-v", "15.0.0", "-c", "{cookie}", "-p", "{port}", "-f", "{filename}"]
+    },
+    
+    "compatibility": {
+        "systems": ["Linux", "Windows", "Mac"],
+        "clients": ["Unity", "Flash"]
+    },
+
+    "isOutdated": false
```
Comment (WiredSpast) [inline store/extensions/PositionGo/extension.json:36]: Missing submissionDate and updateDate
```suggestion
    "submissionDate": "15-09-2022 21:00:00",
    "updateDate": "15-09-2022 21:00:00",
    "isOutdated": false
```

**Review by WiredSpast — CHANGES_REQUESTED** [review]
The zip file must be called `extension.zip` not`PositionGo.zip`

**Review by ghost — COMMENTED** [review]
Resolved done.

**Review by WiredSpast — CHANGES_REQUESTED** [review]
Don't include `fs` in your package.json's dependencies, it's a default lib for which node chooses the best version itself.

Also best to first check if the `locations.json` file exists before trying to read it

image

```js
let positions = [];
if (fs.existsSync("./locations.json")) {
	positions = JSON.parse(fs.readFileSync("./locations.json", "utf8")); // load positions file
}
```

**Comment by WiredSpast** [comment]
You still forgot the `"type": "module"` in the package.json

**Review by ghost — COMMENTED** [review]
Should be good to go now?

**Review by WiredSpast — APPROVED** [review]

File: store/extensions/PositionGo/extension.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    "stable": true,
+    
+    "source": "https://github.com/iUseYahoo/PositionGo",
+    
+    "framework": {
+      "name": "G-Node",
+      "version": "0.2.9"
+    },
+
+    "language": "NodeJS",
```
Comment (WiredSpast) [inline store/extensions/PositionGo/extension.json]:
```suggestion
    "language": "Node.js",
```

**Review by WiredSpast — CHANGES_REQUESTED** [review]

**Review by WiredSpast — APPROVED** [review]

**Pull request merged**